generate/generate.go

```diff
@@ -745,14 +781,49 @@ func hasTerramateHeader(code []byte) bool {
 }
 
 func checkGeneratedFilesConflicts(genfiles []genfile) error {
-	observed := map[string]struct{}{}
+	observed := newStringSet()
```

Comment:
@thiesen now it is explicitely a set since I use it in different places =P

Reply:
:+1: :cool: makes sense!